### PR TITLE
Upgrade to SDK 0.13.52

### PR DESCRIPTION
**Pull request by rohanjr**
Changes call to `daml codegen ts` and name of generated directory in imports.

### CODE REVIEW

File: ui/src/components/LoginScreen.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Button, Form, Grid, Header, Image, Segment } from 'semantic-ui-react'
 import Credentials, { computeToken, makeCredentials } from '../Credentials';
 import Ledger from '@daml/ledger';
-import { User } from '@daml2ts/create-daml-app/lib/create-daml-app/User';
+import { User } from '@daml2ts/create-daml-app/lib/create-daml-app-0.1.0/User';
```
Comment (rohanjr) [inline ui/src/components/LoginScreen.tsx]: @hurryabit @shayne-fletcher Is this how we'd like imports of generated code to look (with the version number in the directory name)?

Comment (hurryabit) [reply ui/src/components/LoginScreen.tsx]: Tbh, I don't know yet how we want this to look. It's way too complicated anyway. We need a solid strategy for this. I wouldn't bother with this for now.

Comment (shayne-fletcher) [reply ui/src/components/LoginScreen.tsx]: I don't know the precise answer to that. However, that's not what we do in DAVL e.g.
```
import * as davl3 from '@daml2ts/davl-v3/lib/davl-v3/DAVL';
import * as davl4 from '@daml2ts/davl-v4/lib/davl-v4/DAVL';
```

Comment (hurryabit) [reply ui/src/components/LoginScreen.tsx]: In DAVL we use `--main-package-name`. Here, we're using the shiny new feature that uses the package name from the manifest in the DAR. @shayne-fletcher you of all people should know what it does. 😉

Comment (rohanjr) [reply ui/src/components/LoginScreen.tsx]: @shayne-fletcher Do you use the main package name inference for codegen in DAVL yet?

Comment (shayne-fletcher) [reply ui/src/components/LoginScreen.tsx]: 🤣

Comment (hurryabit) [reply ui/src/components/LoginScreen.tsx]: @rohanjr No, we don't. We use `--main-package-name`.

Comment (shayne-fletcher) [reply ui/src/components/LoginScreen.tsx]: Right - bit of context @rohanjr. You can invoke `daml2ts` with an optional argument `--<main-package-name>`. The `main-package-name` is the name to associate with the package that contains the main DALF in the DAR e.g. we write `davl-v3` for the name of the folder that will contain `DAVL.ts`. Until yesterday, if not given this argument that would default to the hash of the main package. We do better now by reading the package name from the DAR and using that (if it's there). All "modern" SDKs write a package name into the manifest of a DAR. So what you are seeing there is the package name of the main DALF in the `create-daml-app` DAR (formed obviously from the version number).

Comment (shayne-fletcher) [reply ui/src/components/LoginScreen.tsx]: Wow. So many race conditions in this thread 😄

**Pull request merged**